### DISCUSSION TITLE
[Question] Is it possible to differentiate links to prevent all getting router-link-active by expanded parameter's?

### DISCUSSION THREAD

**Discussion by OpNop** [hannoeru/vite-plugin-pages#373] [Q&A]
Not sure the best way to name this or if this is more of just a general vue-router question, but I have a folder/URL structure that I want to use so that a user is able to manage multiple groups and each group will have the same child navigation just with different ID's for each one. Might be easier to show then explain. 

The folder structure is like
<img width="150" alt="image" src="https://user-images.githubusercontent.com/14283547/216749098-4d0d507d-66eb-42fe-a2b7-dba35d46f71c.png">

and I'm generating the menus with 
```js
  {
    title: themeConfig.guilds[0].name,
    icon: { icon: 'mdi-account-group' },
    children: [
      { 
        title: 'Members',
        icon: { icon: 'mdi-account-group' },
        to: { name: 'guild-id-members', params: { id: themeConfig.guilds[0].guid } },
      },
      { 
        title: 'Logs', 
        icon: { icon: 'mdi-format-list-bulleted-square' }, 
        to: { name: 'guild-id-logs', params: { id: themeConfig.guilds[0].guid } },
      },
    ],
  },
  .... (other guilds here)
```

Links end up like 
```
/guild/0A9D5AFD-9709-E911-81A8-A25FC8B1A2FE/members
/guild/BA7EC8EA-6B52-E811-81A8-90824340DEC8/members
/guild/7D0DB7CC-02FE-E911-81AA-A77AA130EAB8/members
/guild/4EC8BEAF-B669-EB11-81AC-95DFE50946EB/members
and
/guild/0A9D5AFD-9709-E911-81A8-A25FC8B1A2FE/logs
/guild/BA7EC8EA-6B52-E811-81A8-90824340DEC8/logs
/guild/7D0DB7CC-02FE-E911-81AA-A77AA130EAB8/logs
/guild/4EC8BEAF-B669-EB11-81AC-95DFE50946EB/logs
```

but of course because the slug is all the same for the 2 link types they all get marked as active when you go into and one of them like so 
![image](https://user-images.githubusercontent.com/14283547/216749397-f9e15271-df76-4205-87a5-ed20b1c83499.png)

Is this doable? Thanks!

**Comment by marekvospel** [comment]
I belive you can use `router-link-exact-active` class
